src/content/en/updates/2020/08/devtools.md

```diff
@@ -0,0 +1,261 @@
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend.git/+/c55849a7826fcabbae137b0f85cd44c93498581d #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/ca50c1ef5f527bd1e8dd86b53278c7da62d25681 #}
+
+Chromium issue: [772558](https://crbug.com/772558)
+
+
+## Deprecation of “other origins” listing in the Service Workers pane {: #deprecate-sw-other-origins }
+
+DevTools now provides a link to view the full list of service workers from other origins in a new browser tab - [chrome://serviceworker-internals/?devtools](chrome://serviceworker-internals/?devtools).
+
+Previously DevTools displayed a list nested under the Application panel > Service workers.
+
+![Link to other origins](/web/updates/images/2020/08/sw-other-origins.png)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/fbf20a6507edffeab481efedd1e8de710e4cbfbe #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/bb22c1539f01c02985b0b116cebde2143c0e1b75  #}
+
+Chromium issue: [807440](https://crbug.com/807440)
+
+
+## Show coverage summary for filtered items {: #filter-coverage-summary }
+DevTools now recalculates and displays a summary of coverage information dynamically when filters are applied in the Coverage tab. Previously, the Coverage tab always displays a summary of all coverage information. 
+
+The following example shows the coverage summary when no filter applied versus after applying filter by CSS type. 
+
+![Coverage summary for filtered items](/web/updates/images/2020/08/coverage-compare.png)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend.git/+/d6a5f02987283478ffaec6af72a2da59fc552132 #}
+
+Chromium issue: [1061385](https://crbug.com/1090802)
+
+
+## New frame detailed view in Application panel {: #frame-detailed-view }
+DevTools now show a detailed view for each frame. Access it by clicking a frame under the Frames menu in the Application panel.
+
+![New frame detailed view in Application panel](/web/updates/images/2020/08/frame-details.png)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/760462e3a489859865956ea33d1afe5648418a73  #}
+
+Chromium issue: [1093247](https://crbug.com/1093247) 
+
+### Frame details for opened windows {: #pop-up-frame-details }
+
+DevTools now displays opened windows / pop-ups under the frame tree as well. The frame detailed view of the opened windows includes additional security information.
+
+![New frame detailed view in Application panel](/web/updates/images/2020/08/window-opener.png)
+
+More security information will be added to the frame detailed view soon.
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/3b4747f0e2791ad844c93b52c1758208e1e4789c #}
+
+Chromium issue: [1107766](https://crbug.com/1107766) 
+
+
+## Elements and Network panel updates {: #elements-network }
+
+### Capture node screenshots via Elements panel context menu {: #capture-node-screenshot }
```

Comment:
I have seen quite a few requests for this feature so I think it should be bumped up to spot #2